src/components/treePopup/index.tsx

```diff
@@ -145,8 +151,24 @@ const TreePopup: React.FC<TreePopupProps> = ({ popRef, treeInfo }) => {
               </>
             ) : (
               <>
-                <TreeTitle>Open Planting Site</TreeTitle>
-                <CloseIcon onClick={hidePopup} />
+                <>
+                  <TreeTitle>Open Planting Site</TreeTitle>
+                  <CloseIcon onClick={hidePopup} />
+                </>
+                <Line />
+                {!isEmptyString(treeInfo.address) && (
+                  <GreyText strong>Nearby Address</GreyText>
+                )}
+                <GreyText>{treeInfo.address}</GreyText>
```

Comment:
The popup for no tree present and tree present look pretty similar now - could you abstract some of this out?